Tests/test_file_jpeg.py

```diff
@@ -476,17 +476,11 @@ def test_save_correct_modes(self):
             img.save(out, "JPEG")
 
     def test_save_wrong_modes(self):
-        out = BytesIO()
-        for mode in ['LA', 'La', 'RGBa', 'P']:
-            img = Image.new(mode, (20, 20))
-            self.assertRaises(IOError, img.save, out, "JPEG")
-
-    def test_save_modes_with_warnings(self):
         # ref https://github.com/python-pillow/Pillow/issues/2005
```

Comment:
It was in the original commit, before the error was turned into the warning: https://github.com/python-pillow/Pillow/pull/2010/commits/c1da18e0ad541fd3f80de17a766b32e0e032b0ef